examples/basic/console.rb

```diff
@@ -7,12 +7,10 @@
   source "https://rubygems.org"
 
   gem "splunk-otel", path: "../../"
-  gem "opentelemetry-exporter-otlp", "~> 0.21.0"
 end
 
 require "splunk/otel"
 require "splunk/otel/logging"
-require "opentelemetry/exporter/otlp"
```

Comment:
Why is this removed?

Reply:
Don't think the example uses it in any way, `opentelemetry-exporter-otlp` is brought in via the distro anyway

Reply:
Ah right, ok.